lib/site-builder.js

```diff
@@ -213,18 +213,36 @@ SiteBuilder.prototype.readOrWriteConfig = function(configExists) {
 
 SiteBuilder.prototype.jekyllBuild = function() {
   var jekyll = config.jekyll;
-  var args = ['build', '--trace', '--destination', this.buildDestination,
-    '--config', '_config.yml,_config_18f_pages.yml'];
+  var args = [];
 
   if (this.usesBundler) {
     jekyll = config.bundler;
-    args = ['exec', 'jekyll'].concat(args);
+    args = ['exec', 'jekyll'];
+  }
+  args = args.concat(['build', '--trace', '--destination']);
+
+  if (!this.hasInternalConfig) {
+    return this.spawn(jekyll, args.concat(
+      [this.buildDestination,
+       '--config', '_config.yml,' + this.opts.pagesConfig]));
```

Comment:
Good call. I extracted a new `JekyllCommandHelper`, hopefully it makes the logic a little clearer. I held off on the `this.opts.configFiles` bit for now, though I agree it might be something we could support. I'll add it now if you really think it's worth it.